references/classification/utils.py

```diff
@@ -72,3 +72,21 @@ def plot_recorder(lr_recorder, loss_recorder, beta: float = 0.95, **kwargs) -> N
     plt.ylim(vals[min_idx] - 0.1 * delta, max_val + 0.2 * delta)
     plt.grid(True, linestyle="--", axis="x")
     plt.show(**kwargs)
+
+
+class EarlyStopper:
+    def __init__(self, patience: int = 5, min_delta: int = 0):
+        self.patience = patience
+        self.min_delta = min_delta
+        self.counter = 0
+        self.min_validation_loss = float("inf")
+
+    def early_stop(self, validation_loss):
```

Comment:
Missing typings :)

Reply:
Ohhh. I am sorry. That is very silly of me. I did not notice that 🥲